docs/doc.main

```diff
@@ -146,12 +146,20 @@
 
 
   "General" => [
-     "General/styleguide.md",
      "General/other.md",
      "General/faq.md",
-		 "General/SubObjectIterator.md"
   ],
 
   "References" => "references.md",
   "Index" => "manualindex.md",
+  "Developers Corner" => [
```

Comment:
That seems a bit idiosyncratic? How about keeping it simple (and also match the Julia docs) and use
```suggestion
  "Developer Documentation" => [
```

Reply:
From my gut, I feel "References" and "Index" should be last... Tthat's not to say you must change it, I dunno, it's just weird... then again, the Developer docs kinda are separate...

I guess it'd be nice if there was a way to insert a separator into the nav menu, then one could place on before this section (and perhaps another before "References". But I don't think that's available.  Ah well, this is not a reason to hold up this PR further, we can shift things around later.